.circleci/config.yml

```diff
@@ -2,22 +2,48 @@ version: 2
 jobs:
   build:
     docker:
-    - image: circleci/python:3.6.6-node
-    working_directory: ~/singnet/snet-cli
+    - image: ubuntu:latest
```

Comment:
Why it is necessary to install Python and NodeJS manually instead of using prebuilt docker image?